base-server.d.ts

```diff
@@ -0,0 +1,759 @@
+import { Log, Meta, Store, ServerConnection, TestTime } from '@logux/core'
+import { Server as HTTPServer } from 'http'
+
+import { Context, ChannelContext, LoguxPatternParams } from './context'
+import { ServerClient } from './server-client'
+
+export type LoguxMeta = Meta & {
```

Comment:
Let’s rename it to `ServerMeta` to keep name conversion (compare it with `ClientMeta` in `@logux/client`)